src/BuildingRegistry.Api.Legacy.Abstractions/Building/Responses/BuildingSyndicationResponse.cs

```diff
@@ -276,6 +277,12 @@ public class BuildingUnitSyndicationContent
         [DataMember(Name = "IsCompleet", Order = 8)]
         public bool IsComplete { get; set; }
 
+        /// <summary>
+        /// Gebouweenheid afwijking.
+        /// </summary>
+        [DataMember(Name = "AfwijkingVastgesteld", Order = 8)]
```

Comment:
I'm not sure about the description in the summary (I took this from the event property summary), nor about AfwijkingVastgesteld.

Reply:
Waiting for Business.